terraform/terraform.tfvars

```diff
@@ -15,3 +15,5 @@
 gcp_project_id = "<project_id_here>"
 
 memorystore = false
+
+redis-enterprise = false
```

Comment:
```suggestion
redis_enterprise = false
```

Reply:
Resolved in https://github.com/gmflau/microservices-demo/commit/94ee7736dae7a841face746f3dfc0c635c260150.